### PR TITLE
CONSOLE-3788: migrate console Protractor tests to Cypress

### PR DESCRIPTION
**Pull request by rhamilto**

### CODE REVIEW

**Comment by rhamilto** [comment]
Work is tech debt and has no customer-facing impact.

/label docs-approved
/label px-approved
/label qe-approved

File: frontend/packages/integration-tests-cypress/tests/cluster-settings/alertmanager/receivers/email.cy.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+const warning = 'warning';
+const emailTo = 'you@there.com';
+const emailFrom = 'me@here.com';
+const emailSmarthost = 'smarthost:8080';
+const username = 'username';
+const password = 'password';
+const identity = 'identity';
+const secret = 'secret';
+const html = 'myhtml';
+
```
[the remark this answers is not in the document]
Comment (rhamilto) [reply frontend/packages/integration-tests-cypress/tests/cluster-settings/alertmanager/receivers/email.cy.ts:20]: Good point.

**Review by jhadvig — APPROVED** [review]
Kudos @rhamilto for this big change 🙌 
/lgtm
/approve

**Pull request merged**